truss-chains/truss_chains/model_skeleton.py

```diff
@@ -15,7 +16,11 @@ class TrussChainletModel:
     _chainlet: definitions.ABCChainlet
 
     def __init__(
-        self, config: dict, data_dir: pathlib.Path, secrets: secrets_resolver.Secrets
+        self,
+        config: dict,
+        data_dir: pathlib.Path,
+        secrets: secrets_resolver.Secrets,
+        environment: Optional[dict] = None,
```

Comment:
Please add a TODO to remove this once truss CLI and context builder are in sync again.